sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRepositoryLiveTests.cs

```diff
@@ -0,0 +1,277 @@
+                    CanRead = false,
+                    CanWrite = false,
+                    CanDelete = false,
+                });
+
+            // Assert
+            Assert.IsFalse(properties.WriteableProperties.CanList);
+            Assert.IsFalse(properties.WriteableProperties.CanRead);
+            Assert.IsFalse(properties.WriteableProperties.CanWrite);
+            Assert.IsFalse(properties.WriteableProperties.CanDelete);
+
+            RepositoryProperties updatedProperties = await repository.GetPropertiesAsync();
+
+            Assert.IsFalse(updatedProperties.WriteableProperties.CanList);
+            Assert.IsFalse(updatedProperties.WriteableProperties.CanRead);
+            Assert.IsFalse(updatedProperties.WriteableProperties.CanWrite);
+            Assert.IsFalse(updatedProperties.WriteableProperties.CanDelete);
+
+            // Cleanup
+            await repository.SetPropertiesAsync(originalContentProperties);
+        }
+
+        [RecordedTest, NonParallelizable]
+        public async Task CanDeleteRepository()
+        {
+            // Arrange
+            List<string> tags = new List<string>()
+            {
+                "latest",
+                "v1",
+                "v2",
+                "v3",
+                "v4",
+            };
+
+            var client = CreateClient();
+            var repository = client.GetRepository(_repositoryName);
+
+            try
+            {
+                if (Mode != RecordedTestMode.Playback)
+                {
+                    await ImportImage(_repositoryName, tags);
+                }
+
+                // Act
+                await repository.DeleteAsync();
+
+                // This will be removed, pending investigation into potential race condition.
+                // https://github.com/azure/azure-sdk-for-net/issues/19699
+                if (Mode != RecordedTestMode.Playback)
+                {
+                    await Task.Delay(5000);
+                }
+
+                // Assert
+                Assert.ThrowsAsync<RequestFailedException>(async () => { await repository.GetPropertiesAsync(); });
+            }
+            finally
+            {
+                // Clean up - put the repository with tags back.
+                if (Mode != RecordedTestMode.Playback)
+                {
+                    await ImportImage(_repositoryName, tags);
+                }
+            }
+        }
+
+        [RecordedTest]
+        public async Task CanGetManifests()
+        {
+            // Arrange
+            var client = CreateClient();
+            var repository = client.GetRepository(_repositoryName);
+
+            // Act
+            AsyncPageable<ArtifactManifestProperties> manifests = repository.GetManifestsAsync();
+
+            ArtifactManifestProperties latest = null;
+            await foreach (ArtifactManifestProperties manifest in manifests)
+            {
+                if (manifest.Tags.Count > 0 && manifest.Tags.Contains("latest"))
```

Comment:
Is checking for "latest" a real scenario outside of tests? If so, maybe it would be worth adding some kind of helper method like `IsLatestTag(string tagName)`

Reply:
I worry about the ambiguity in a name like that - does it mean the last tag I updated or the tag I'm calling "latest" ?  I'll let customer feedback drive the addition of convenience methods like this.  Thanks for the suggestion!